exercises/strings/rot13/rot13.js

```diff
@@ -31,9 +31,16 @@
 function rot13(string) {
   // This is your job. :)
 }
+let sourceAlphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Not exactly working yet. Here's some tips: 
    1. If you going to compute using the uppercase letters, you should convert the input string into all uppercase so that the code will work correctly for all letters in the strings. Because the input could be `hElLo`. There's a simple function that does this.
    2. You can access a specific letter in a string using `.charAt(index)`